tox.ini

```diff
@@ -398,51 +176,21 @@ commands_pre =
   pip install -e {toxinidir}/opentelemetry-api \
               -e {toxinidir}/opentelemetry-sdk \
               -e {toxinidir}/opentelemetry-instrumentation \
-              -e {toxinidir}/instrumentation/opentelemetry-instrumentation-requests \
-              -e {toxinidir}/instrumentation/opentelemetry-instrumentation-wsgi \
-              -e {toxinidir}/instrumentation/opentelemetry-instrumentation-flask
+              -e {toxinidir}/opentelemetry-python-contrib/instrumentation/opentelemetry-instrumentation-requests \
+              -e {toxinidir}/opentelemetry-python-contrib/instrumentation/opentelemetry-instrumentation-wsgi
 
 commands =
   {toxinidir}/scripts/tracecontext-integration-test.sh
 
 [testenv:docker-tests]
 deps =
   pytest
-  asyncpg==0.20.1
-  docker-compose >= 1.25.2
-  mysql-connector-python ~=  8.0
-  pymongo ~= 3.1
-  pymysql ~= 0.9.3
-  psycopg2-binary ~= 2.8.4
-  aiopg >= 0.13.0
-  sqlalchemy ~= 1.3.16
-  redis ~= 3.3.11
-  celery ~= 4.0, != 4.4.4
 
 changedir =
   tests/opentelemetry-docker-tests/tests
 
 commands_pre =
-  pip install -e {toxinidir}/opentelemetry-api \
-              -e {toxinidir}/opentelemetry-sdk \
-              -e {toxinidir}/opentelemetry-instrumentation \
-              -e {toxinidir}/tests/util \
-              -e {toxinidir}/instrumentation/opentelemetry-instrumentation-asyncpg \
-              -e {toxinidir}/instrumentation/opentelemetry-instrumentation-celery \
-              -e {toxinidir}/instrumentation/opentelemetry-instrumentation-dbapi \
-              -e {toxinidir}/instrumentation/opentelemetry-instrumentation-mysql \
-              -e {toxinidir}/instrumentation/opentelemetry-instrumentation-psycopg2 \
-              -e {toxinidir}/instrumentation/opentelemetry-instrumentation-pymongo \
-              -e {toxinidir}/instrumentation/opentelemetry-instrumentation-pymysql \
-              -e {toxinidir}/instrumentation/opentelemetry-instrumentation-sqlalchemy \
-              -e {toxinidir}/instrumentation/opentelemetry-instrumentation-aiopg \
-              -e {toxinidir}/instrumentation/opentelemetry-instrumentation-redis \
-              -e {toxinidir}/instrumentation/opentelemetry-instrumentation-system-metrics \
-              -e {toxinidir}/exporter/opentelemetry-exporter-opencensus
-  docker-compose up -d
-  python check_availability.py
+  pip install -e {toxinidir}/exporter/opentelemetry-exporter-opencensus
```

Comment:
are the api/sdk packages no longer needed here?

Reply:
we should also still have docker-compose up here to launch the opencensus collector

Reply:
When I looked at the `exporter/opencensus` tests it didn't seem to be using the `docker-compose`, but I haven't verified completely if that's the case.

Reply:
You were right, updated this part of `tox.ini`! 😄